S/SCIP/build_tarballs.jl

```diff
@@ -48,9 +48,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
-    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2")
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065")),
+    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d", version=v"6.1.2")),
```

Comment:
Ok, I just merged but I don't understand this

Reply:
I hadn't noticed the reordering. I had an error in the constructor Dependency(PkgSpec, version), so I integrated the version requirement in the PkgSpec Constructor

Reply:
Well, if you didn't do that on purpose I believe that was wrong

Reply:
> I had an error in the constructor Dependency(PkgSpec, version)

Did you have it locally?

Reply:
Yes:
```
julia> dependencies = [
           Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
           Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2")
           Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
       ]
ERROR: MethodError: no method matching Dependency(::Pkg.Types.PackageSpec, ::VersionNumber)
Closest candidates are:
  Dependency(::Pkg.Types.PackageSpec) at /home/mbesancon/.julia/packages/BinaryBuilderBase/fylAz/src/Dependencies.jl:29
  Dependency(::Any) at /home/mbesancon/.julia/packages/BinaryBuilderBase/fylAz/src/Dependencies.jl:29
Stacktrace:
 [1] top-level scope at REPL[12]:1

julia> dependencies = [
           Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
           Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d", version=v"6.1.2"))
           Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
       ]
3-element Array{Dependency,1}:
 Dependency(PackageSpec(
  name = bliss_jll
  uuid = 508c9074-7a14-5c94-9582-3d4bc1871065
  version = *
))
 Dependency(PackageSpec(
  name = GMP_jll
  uuid = 781609d7-10c4-51f6-84f2-b8444358ff6d
  version = VersionSpec("6.1.2")
))
 Dependency(PackageSpec(
  name = CompilerSupportLibraries_jll
  uuid = e66e0078-7015-5450-92f7-15fbd957f2ae
  version = *
))
```

Reply:
Julia v1.5 I guess?

Reply:
Anyway, this line is wrong, need to bring it back as it was before

Reply:
> Julia v1.5 I guess?

Yes, I am not part of the cool kis running on a bleeding master